script/mockbigscreenplayer.js

```diff
@@ -143,7 +150,7 @@ define('bigscreenplayer/mockbigscreenplayer',
         mediaKind = bigscreenPlayerData && bigscreenPlayerData.media && bigscreenPlayerData.media.kind || 'video';
         windowType = newWindowType || WindowTypes.STATIC;
         subtitlesAvailable = true;
-        subtitlesEnabled = false;
+        subtitlesEnabled = enableSubtitles;
```

Comment:
previously the mock ignored `enableSubtitles`. Was there a reason to ignore it?

Reply:
Yep, it wasn't exercised through the mock.